src/pages/media-kit/index.module.scss

```diff
@@ -0,0 +1,177 @@
+.container {
+  position: relative;
+  box-sizing: border-box;
+  width: 1440px;
+  max-width: 1440px;
+  margin: 0 auto;
+  padding: 0 120px;
+
+  @media screen and (max-width: 1670px) {
+    overflow-x: hidden;
+  }
+
+  @media screen and (max-width: 1440px) {
+    max-width: 100vw;
+  }
+  @media screen and (max-width: 750px) {
```

Comment:
It looks like this file hasn't been processed by `lint-staged`.

Reply:
I have dry-run `prettier` locally but it didn't modify this

Reply:
`stylelint` does modification.

I've tested by adding a blank line and found that `lint-staged` didn't work as expected. We may should put an eye on it.

```bash
➜  nervos-official-website git:(add-media-kit) ✗ gst
On branch add-media-kit
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   src/pages/media-kit/index.module.scss

no changes added to commit (use "git add" and/or "git commit -a")
➜  nervos-official-website git:(add-media-kit) ✗ ga .
➜  nervos-official-website git:(add-media-kit) ✗  git commit
[add-media-kit eaed304] wip
 1 file changed, 1 insertion(+)
➜  nervos-official-website git:(add-media-kit)
```

I will commit fixes of stylelint soon

Reply:
Formatted by https://github.com/Magickbase/nervos-official-website/pull/359/commits/7dcfa69908ff30351edb9e32721fd5b26595ce24

Reply:
> I've tested by adding a blank line and found that `lint-staged` didn't work as expected. We may should put an eye on it.

Fixed by https://github.com/Magickbase/nervos-official-website/pull/363